render-student.js

```diff
@@ -1,14 +1,26 @@
+function renderData(students) {
+    for (let student of students) {
```

Comment:
This is not a good solution since it's an unnecessary loop.
Look for my next comment for another solution.